submissions/madmaxWMFU/frogger-game/js/app.js

```diff
@@ -0,0 +1,95 @@
+const playerStartValue = {
+    "x": 200,
+    "y": 400,
+    "icon": "images/char-boy.png"
+};
+
+let score = document.querySelector(".user-score");
+
+class Enemy {
+    constructor(x, y, speed) {
+        this.x = x;
```

Comment:
`Enemy` and `Player` have a common properties (`x, y, sprite`). In OOP better way save it in a common class. IMO `Character` class will be good solution.